api/src/main/java/org/openmrs/module/datafilter/impl/api/DataFilterDefaultResponse.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.datafilter.impl.api;
+
+import java.util.Date;
+import java.util.Map;
+
+public class DataFilterDefaultResponse {
```

Comment:
Please build atop the REST web services module’s API to maintain uniformity from a client perspective.